Earthfile

```diff
@@ -152,12 +152,16 @@ go-build:
   ARG GOFLAGS="-ldflags=-X=github.com/crossplane/crossplane/internal/version.version=${CROSSPLANE_VERSION}"
   ARG CGO_ENABLED=0
   FROM +go-modules
+  LET extension = ""
```

Comment:
super nit: what do you think about calling this `ext` just to make it shorter?

Reply:
sure, done, I was juggling between naming it "extensions" or "ext" actually 😸